baker/GrapherImageBaker.tsx

```diff
@@ -38,27 +38,39 @@ export async function bakeGraphersToPngs(
     ])
 }
 
-async function getGraphersAndRedirectsBySlug() {
+export async function getGraphersAndRedirectsBySlug() {
     const { graphersBySlug, graphersById } = await getPublishedGraphersBySlug()
 
     const redirectQuery = db.queryMysql(
         `SELECT slug, chart_id FROM chart_slug_redirects`
     )
 
     for (const row of await redirectQuery) {
-        graphersBySlug.set(row.slug, graphersById.get(row.chart_id))
+        const grapher = graphersById.get(row.chart_id)
+        if (grapher !== undefined) {
+            graphersBySlug.set(row.slug, grapher)
+        }
     }
 
     return graphersBySlug
 }
 
-export async function getPublishedGraphersBySlug() {
+export async function getPublishedGraphersBySlug(
+    includePrivate: boolean = true
```

Comment:
Seems like the default should be `false`, otherwise you're now baking private graphers into the site.